azure/LongRunningOperations.md

```diff
@@ -0,0 +1,69 @@
+# Long-Running Operations
```

Comment:
I made this a separate document for now -- I figured we could work out what the content should be and then decide where it should live.

Reply:
This doc & content look good. One thing we may want to call out is that this is a REST binding to LROs and that implementations for other protocols should follow the same pattern. For example, in resource based pattern, you get a "shallow" representation of the resource. In stepwise, you get an alternate location to go ask for the state of the resource. 

Do we want to add anything about the identity of the resource in the two different models? For example, in resource oriented LROs, you can get back the shallow representation of the resource, which should have its ID. This would not be the case in stepwise.

What about idempotency? Should we explicitly call out that these are idempotent operations?